src/test/regress/expected/columnar_vacuum_vs_insert.out

```diff
@@ -49,14 +49,11 @@ step s1-insert:
     INSERT INTO test_vacuum_vs_insert SELECT i, 2 * i FROM generate_series(1, 3) i;
 
 step s2-vacuum-full:
-    VACUUM FULL VERBOSE test_vacuum_vs_insert;
+    VACUUM FULL test_vacuum_vs_insert;
  <waiting ...>
 step s1-commit: 
     COMMIT;
 
```

Comment:
@jeff-davis is the INFO/DETAIL critical to keep for this test?

Reply:
I suggest keeping `VERBOSE` and adding an alternate expected output file. We probably want to add more details here in the future.

Reply:
@jeff-davis thanks for your input.
I added a normalization rule which will help if we have other such cases
`s/".*\.(.*)": (found .* removable)/"\1": \2/g`

Closing this PR